.github/actions/install-dependencies/action.yml

```diff
@@ -15,11 +15,22 @@ runs:
         sudo apt-get install -y \
           librsvg2-bin \
           ghostscript \
-          graphviz \
           pkg-config \
           libgraphviz-dev \
           gsfonts \
           libfreetype6-dev \
           libfontconfig1-dev
       # Explanation of some of these dependencies:
       # https://github.com/ImageMagick/ImageMagick/issues/374#issuecomment-279252866
+
+    - uses: conda-incubator/setup-miniconda@35d1405e78aa3f784fe3ce9a2eb378d5eeb62169 # v2.1.1
+      with:
+        channels: conda-forge
+        python-version: ${{ matrix.python-version || '' }}
+        miniforge-version: latest
+
+    # use login shell for conda activation
+    - shell: bash -leo pipefail {0}
```

Comment:
There is an `activate-environment:` configuration which presumably handles this?

Reply:
`activate-environment` is set to `test` by default - the `test` env gets activated for every step. However, the default shell is `bash --noprofile --norc -eo pipefail {0}` which prevents this activation. So you have to use a login shell instead.

See https://github.com/conda-incubator/setup-miniconda#important